docs/geedocs/answer/2987876.html

```diff
@@ -45,6 +61,7 @@
 <li>MrSID</li>
```

Comment:
MrSid binaries are still shipped in 5.2.0 ?

Reply:
No, it isn't, and neither is Kakadu. I went back and forth on this for a bit. The first sentence is: "Open Google Earth Enterprise may utilize the following products as integrated components, linked libraries, or optional modules." Under that definition it seems that MrSID would still apply, but it wouldn't necessarily be provided. So, I can either change the sentence to exclude products that aren't bundled, indicate products that would need to be provided separately (maybe with *), or just leave it as is. Do you have a preference?

Reply:
I'd suggest changing it to only mention the things that are included with GEE.  I don't think we need to say anything about stuff users add themselves.

Reply:
Since we don't bundle nor ship them, it seems to me, that we would not need to mention them. It would be up to the end user if they add extra components to ensure that they are in compliance with any licensing requirements. It may be confusing to list them as GDAL for example has its license mentioned some where else. Also it's possible to include other libraries that we don't know about. I would probably not listed them or otherwise list only the one that we don't bundle and mention that the list if only listing some of the modules not all of them. My preference is to not list them.